app/modules/entities/components/layouts/section_label.svelte

```diff
@@ -0,0 +1,59 @@
+<script>
+  import { i18n } from '#user/lib/i18n'
+  import { loadInternalLink } from '#lib/utils'
+  import { isPropertyUri } from '#lib/boolean_tests'
+
+  export let property, label, entitiesLength, filteredEntitiesLength, uri
+</script>
+{#if label}
+  <div class="label-wrapper">
+    <div class="left-section">
+      <h3>
+        {label}
+      </h3>
+      {#if entitiesLength > 0}
+        <span class="counter">
+          {#if filteredEntitiesLength >= 0 && filteredEntitiesLength !== entitiesLength}
+            {filteredEntitiesLength} /
+          {/if}
+          {entitiesLength}
+        </span>
+      {/if}
+    </div>
+    {#if isPropertyUri(property)}
+      <a
+        href={`/entity/${property}-${uri}`}
+        on:click={loadInternalLink}
+      >
+        {i18n('Advanced options')}
```

Comment:
We already use this term in the inventory browser for a button where you simply unfold the advanced browsing options menu, rather than load a new page. Could we rephrase that to give the idea that we will change page? Something like "Open in advanced list browser"?

Reply:
it was to not have another i18n key, but yes im fine with the suggestion. Done in a3c874456